content/includes/footer.html

```diff
@@ -4,23 +4,23 @@
     <div class="row va-flex">
     <ul class="va-footer-linkgroup" id="footer-services">
       <li><h4 class="va-footer-linkgroup-title">Services and Benefits</h4></li>
-      <li><a href="https://www.vets.gov/disability-benefits/">Disability Benefits</a></li>
-      <li><a href="https://www.vets.gov/health-care/">Health Care Benefits</a></li>
-      <li><a href="https://www.vets.gov/education/">Education Benefits</a></li>
-      <li><a href="https://www.vets.gov/housing-assistance/">Housing Assistance</a></li>
-      <li><a href="https://www.vets.gov/employment/">Careers and Employment</a></li>
-      <li><a href="https://www.vets.gov/life-insurance/">Life Insurance</a></li>
-      <li><a href="https://www.vets.gov/pension/">Pension Benefits</a></li>
-      <li><a href="https://www.vets.gov/burials-and-memorials">Burials and Memorials</a></li>
+      <li><a href="https://www.vets.gov/disability-benefits/"><span class="usa-sr-only"></span>Disability Benefits</a></li>
+      <li><a href="https://www.vets.gov/health-care/"><span class="usa-sr-only"></span>Health Care Benefits</a></li>
+      <li><a href="https://www.vets.gov/education/"><span class="usa-sr-only"></span>Education Benefits</a></li>
+      <li><a href="https://www.vets.gov/housing-assistance/"><span class="usa-sr-only"></span>Housing Assistance</a></li>
+      <li><a href="https://www.vets.gov/employment/"><span class="usa-sr-only"></span>Careers and Employment</a></li>
+      <li><a href="https://www.vets.gov/life-insurance/"><span class="usa-sr-only"></span>Life Insurance</a></li>
+      <li><a href="https://www.vets.gov/pension/"><span class="usa-sr-only"></span>Pension Benefits</a></li>
+      <li><a href="https://www.vets.gov/burials-and-memorials"><span class="usa-sr-only"></span>Burials and Memorials</a></li>
```

Comment:
Could you clarify why we're adding empty `<span>`s?

`.usa-sr-only` does the following (from `node_modules/uswds/dist/css/uswds.css` line 823):
```css
.usa-sr-only {
  position: absolute;
  left: -999em;
}
```
It looks like this is used for adding content that we want to be read off (and and interacted with) by screen readers but not shown visually. Is the plan to add "External link" to all of them? If so, @markgreenburg's [PR](https://github.com/department-of-veterans-affairs/vets-website/pull/6863) may already take care of all this.

Reply:
You are correct @cvalarida, the plan is to add "External Link" here pending input on @markgreenburg's PR, where I have asked if that is the appropriate text. As for the correct approach, @markgreenburg and I are exploring that in [this](https://github.com/department-of-veterans-affairs/vets-website/pull/6581#discussion_r153851797) thread.

Reply:
Okay, cool. Just wanted to make sure. Thanks! 😄